### PR TITLE
feat: allow to edit commit message in $EDITOR

### LINKED ISSUE #14: allow to manually edit resulted commit message
I think we could pass resulted commit message to `git commit -m`

### PR DESCRIPTION
**Pull request by Guria**
resolves #14

### CODE REVIEW

**Comment by Guria** [comment]
@leonardoanalista check this out

**Comment by leonardoanalista** [comment]
I just looked at the module `editor` source code https://github.com/substack/node-editor/blob/master/index.js

It tries in this order: `opts.editor` or `process.env.EDITOR`. Finally it defaults to `vim` or `notepad` depending on the platform.

It is all good! thanks

**Comment by Guria** [comment]
Ok. I will try to finish it today.

**Comment by leonardoanalista** [comment]
no rush. Cheers
Leo

File: spec/czCustomizableSpec.js [hunk 1]
```diff
@@ -152,6 +152,42 @@ describe('cz-customizable', function() {
...
+
+    var answers = {
+      confirmCommit: 'edit',
+      type: 'feat',
+      subject: 'create a new cool feature'
+    };
+
+    commitAnswers(answers);
+    setTimeout(function() {
+      expect(commit.wasCalled).toEqual(false);
```
Comment (leonardoanalista) [inline spec/czCustomizableSpec.js]: where did the `wasCalled` come from?

File: spec/czCustomizableSpec.js [hunk 2]
```diff
@@ -152,6 +152,42 @@ describe('cz-customizable', function() {
     expect(commit).toHaveBeenCalledWith('WIP: this is my work-in-progress');
   });
 
+  it('should allow edit message before commit', function(done) {
```
Comment (leonardoanalista) [inline spec/czCustomizableSpec.js]: Tests look small and easy to read

File: spec/czCustomizableSpec.js (hunk 1 above)
Comment (Guria) [inline spec/czCustomizableSpec.js]: `jasmine-node` api seems slightly different from original `jasmine`.
I have looked up it in `jasmine-node` specs. Haven't found any other documentation.

File: spec/czCustomizableSpec.js (hunk 2 above)
Comment (Guria) [inline spec/czCustomizableSpec.js]: We can use something like `sed` to change message, but I have decided to keep it simple.

**Comment by Guria** [comment]
Wait, do not merge. I forgot to update README

**Comment by Guria** [comment]
Oh. Not this one PR but #20 
Both ready now.
If both OK for you, just merge #20 because it includes commit from #19. Both would be marked as merged

**Comment by leonardoanalista** [comment]
it looks good! I found the `wasCalled` in the jasmine-node source code. API is a bit diff.
You should be able to merge PR's now. Let me know if you can't.

**Comment by Guria** [comment]
I can.

**Pull request merged**